packages/user/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "tsconfig/base.json",
+  "exclude": ["src/**/__test__/**/*"],
+  "compilerOptions": {
```

Comment:
make this file consistant with other packages

Reply:
It is consistent with other packages those having  tests  with them.